public/images/common/update.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.1926 2.27931C13.6476 2.16431 13.1236 2.52331 13.0156 3.06331L12.7586 4.35031C11.6496 2.89931 9.92059 2.00031 8.00459 2.00031C5.71059 2.00031 3.58659 3.33631 2.59259 5.40331C2.35359 5.90131 2.56359 6.49931 3.06159 6.73831C3.56259 6.97931 4.15759 6.76631 4.39559 6.26931C5.05759 4.89031 6.47459 4.00031 8.00459 4.00031C9.31459 4.00031 10.4986 4.62631 11.2336 5.63731L9.72959 5.44531C9.17959 5.38631 8.67959 5.76231 8.61059 6.31031C8.54159 6.8583 8.92759 7.3593 9.47559 7.42931L13.1406 7.8983C13.1836 7.90331 13.2266 7.90631 13.2686 7.90631C13.7386 7.90631 14.1536 7.57431 14.2486 7.10231L14.9766 3.45631C15.0856 2.91431 14.7336 2.38731 14.1926 2.27931Z" fill="currentColor"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12.9435 9.27051C12.4485 9.03151 11.8495 9.23951 11.6085 9.73651C10.9445 11.1115 9.52952 11.9995 8.00452 11.9995C6.66052 11.9995 5.45352 11.3405 4.72052 10.2845L6.24752 10.4645C6.79752 10.5345 7.29252 10.1375 7.35752 9.58851C7.42252 9.04151 7.03052 8.54251 6.48152 8.47851L2.84652 8.04851C2.33852 7.98751 1.85152 8.33451 1.74952 8.84551L1.01952 12.4925C0.911521 13.0345 1.26252 13.5605 1.80352 13.6685C1.87052 13.6825 1.93552 13.6885 2.00152 13.6885C2.46752 13.6885 2.88552 13.3605 2.98052 12.8845L3.23252 11.6295C4.34052 13.0935 6.07852 13.9995 8.00452 13.9995C10.2915 13.9995 12.4135 12.6675 13.4095 10.6055C13.6495 10.1085 13.4415 9.51051 12.9435 9.27051Z" fill="currentColor"/>
+</svg>
```

Comment:
Can we use the Autorenew icon from MUI instead?

Reply:
We are moving away from MUI icons AFAIK but cc @liliiaorlenko

Reply:
yes we have our own library so pls align with it https://www.figma.com/file/hsPhJR6ErHx0O4j71E9Dd5/Icon-Library?node-id=1008%3A223&t=7zTIwetwK0nXHQJ3-1

Reply:
The button containing the icon was removed